### ISSUE TITLE
Code Climate - Ignore Certain "Similar Blocks of Code" Refactoring Suggestions

### ISSUE THREAD

**Issue by mjoyce91** [18F/State-TalentMAP#1023]
Some Code Climate refactoring suggestions are a bit strict, such as ones like this:

<img width="850" alt="screen shot 2017-11-27 at 3 32 32 pm" src="https://user-images.githubusercontent.com/11576347/33287960-8a1f3694-d388-11e7-8b09-cb03d77c0337.png">

Unless there is a way to simplify this, we should consider reducing the strictness of this rule, if possible.

@burgwyn @jseppi

**Comment by jseppi** [comment]
Yeah, I am fine with reducing the strictness or removing that rule.

**Comment by burgwyn** [comment]
Let's disable the duplication engine.  @mjoyce91 - can you update #1022 with
```
  duplication:
    enabled: false
```
in `.codeclimate.yml`
